core/tee/tee_cryp_utl.c

```diff
@@ -378,6 +379,20 @@ TEE_Result tee_prng_add_entropy(const uint8_t *in, size_t len)
 	return TEE_SUCCESS;
 }
 
+/*
+ * override this in your platform code to feed the PRNG
+ * platform-specific jitter entropy.  This implementation
+ * does not efficiently deliver entropy and is here for
+ * backwards-compatibility.
+ */
```

Comment:
Please format this comment better:
- Sentences should start with an uppercase letter
- No double space
- Standard line width is 80 characters (applies to other comments as well)